data-serving/data-service/test/case.test.ts

```diff
@@ -1,5 +1,30 @@
 import request from 'supertest';
 import app from '../src/index';
+import mongoose from 'mongoose';
+import { Case } from '../src/model/case';
+
+beforeAll(async () => {
+    await mongoose.connect(
+        // This is provided by jest-mongodb.
+        // The `else testurl` is to appease Typescript.
+        process.env.MONGO_URL || 'testurl',
+        { useNewUrlParser: true, useCreateIndex: true },
+        (err) => {
+            if (err) {
+                console.error(err);
+                process.exit(1);
```

Comment:
Looking at https://jest-bot.github.io/jest/docs/setup-teardown.html#one-time-setup makes me think that if beforeAll returns promise (could just be return mongoose.connect() in this case) then you don't need to check for the result and tests would fail if that promise isn't resolved? I might be wrong but please double check as it would simplify the setup.

Reply:
yeah I confirmed in my PR that the test can simply be:

```
beforeAll(() => {
    return mongoose.connect(
        // This is provided by jest-mongodb.
        // The `else testurl` is to appease Typescript.
        process.env.MONGO_URL || 'testurl',
        { useNewUrlParser: true, useUnifiedTopology: true },
    );
});

afterAll(() => {
    return mongoose.disconnect();
});
```

Reply:
Nice find. Updated, thanks.